source/accessibility.html.erb

```diff
@@ -30,33 +18,33 @@ layout: govuk
   <li>use most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
 </ul>
 
-<h2 class="govuk-heading-m">How we tested this website</h2>
+<h2 class="govuk-heading-m">Compliance status</h2>
 
-<p>The GOV.UK Forms website was last tested on <time datetime="2022-06-08">8 June 2022</time>. The test was carried out by GDS.</p>
+<p>GDS tested the GOV.UK Forms website on <time datetime="2022-06-08">8 June 2022</time>.</p>
 
-<p>GDS assessed the GOV.UK Forms website against the Web Content Accessibility Guidelines WCAG 2.1.</p>
+<p>This website is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard.</p>
 
-<h2 class="govuk-heading-m">What to do if you have difficulty using this website</h2>
 
-<p>If you have difficulty using this website, contact us by sending an email to <a href="mailto:govuk-forms@digital.cabinet-office.gov.uk">govuk-forms@digital.cabinet-office.gov.uk</a>.</p>
-
-<h2 class="govuk-heading-m">Reporting accessibility problems</h2>
+<h2 class="govuk-heading-m">Feedback and contact information</h2>
 
 <p>We’re always looking to improve the accessibility of this website.</p>
 
 <p>If you find any problems, or think we’re not meeting accessibility requirements, email <a href="mailto:govuk-forms@digital.cabinet-office.gov.uk">govuk-forms@digital.cabinet-office.gov.uk</a>.</p>
 
-<h2 class="govuk-heading-m">If you’re not happy with our response</h2>
+
+<h2 class="govuk-heading-m">Enforcement procedure</h2>
 
 <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).</p>
 
-<p>If you are not happy with how we respond to your complaint, contact the <a rel="external" href="https://www.equalityadvisoryservice.com/">Equality Advisory and Support Service</a>, which is run on behalf of EHRC.</p>
+<p>If you’re not happy with how we respond to your complaint, contact the <a rel="external" href="https://www.equalityadvisoryservice.com/">Equality Advisory and Support Service</a>, which is run on behalf of EHRC.</p>
 
 <h2 class="govuk-heading-m">Technical information about this website’s accessibility</h2>
 
 <p>GDS is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
```

Comment:
Think we can delete this "Technical information about this website’s accessibility" heading and sentence as it's been added at the top?